docs/using-latest-daily.md

```diff
@@ -1,6 +1,6 @@
 # Set up your machine to use the latest Aspire builds
 
-These instructions will get you set up with the latest build of Aspire. If you just want the last preview release of .NET Aspire, the packages are on nuget.org, and install the latest [Visual Studio 2022 version 17.9 Preview](https://visualstudio.microsoft.com/vs/preview/) for the tooling.
+These instructions will get you set up with the latest build of Aspire. If you just want the last preview release of .NET Aspire, the packages are on nuget.org, and install the latest [Visual Studio 2022 version 17.11 Preview](https://visualstudio.microsoft.com/vs/preview/) for the tooling.
```

Comment:
Same here, not sure if we should be making the claim that 17.11 is required.

Reply:
Given the ties between DCP and VS, any new feature that involves VS behavior is by definition going to require the preview bits, even if building the repo doesn't. I figured it was just easier to say that preview is required.

Reply:
I just think that there are still some customers out there that have more strict limitations on which versions of tooling they can use for their projects, so I'd probably lean towards saying that the minimum requirement is 17.10 and 17.11 encouraged, until we find that 17.10 is actually not enough. Just my 2 cents though.